packages/mobile/ios/Podfile.lock

```diff
@@ -1202,4 +1202,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 6927948487d04eb4c86fe5056d0c6dc0bd531bf3
 
-COCOAPODS: 1.11.3
+COCOAPODS: 1.12.0
```

Comment:
Intentional?

Reply:
Might wanna check with client team before checking this in.

Reply:
cc @sliptype . I believe we discussed this a few days ago and your answer was along the lines of "should be fine as long as you can run the mobile app" 😁

Reply:
Yup I have this change locally too! Commit it 💪